### PR TITLE
chore(GAE flex): rm unused sendgrid samples

### PR DESCRIPTION
**Pull request by ace-n**
@qchong please LGTM, thanks!

@engelke FYI

### CODE REVIEW

**Review by engelke — APPROVED** [review]

**Comment by qchong** [comment]
LGTM

**Comment by qchong** [comment]
LGTM

**Comment by ace-n** [comment]
Test failures seem unrelated, re-running...

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
You need to remove the project from the solution file that's on appengine/flexible as well.
After that LGTM and if there are unrelated failures I can force merge.

**Comment by ace-n** [comment]
Failures seem unrelated. @amanda-tarafa I'll rerun the tests one more time, and let you merge if that doesn't help.

File: appengine/flexible/AppEngineFlex.sln [hunk 1]
```diff
@@ -8,8 +8,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Analytics", "Analytics\Anal
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RedisCache", "RedisCache\RedisCache.csproj", "{6E430843-E06E-47C5-ADC2-475C8EDE81DB}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SendGrid", "SendGrid\SendGrid.csproj", "{0C33CDB9-DC3C-4832-AA81-83FE5CAFEF57}"
```
Comment (amanda-tarafa) [inline appengine/flexible/AppEngineFlex.sln:11]: You also need to remove the configuration associated to this project, basically entries that reference 0C33CDB9-DC3C-4832-AA81-83FE5CAFEF57. I gather you are not using Visual Sutdion?

**Review by amanda-tarafa — COMMENTED** [review]
Still some pieces left on the solution file.
I can fetch the PR and do the .sln changes on Visual Studion if you don't use it. It's the best way to make sure that the file ends up as it should. Are you OK with that?

File: appengine/flexible/AppEngineFlex.sln (hunk 1 above)
Comment (ace-n) [reply appengine/flexible/AppEngineFlex.sln:11]: Nope - I guess that would explain it.

If you can make the changes needed, that'd be much appreciated! 🙂

**Review by amanda-tarafa — APPROVED** [review]
Done. LGTM

**Pull request merged**